mpitests/test_adjoint_jacobian.py

```diff
@@ -936,7 +934,7 @@ def circuit(params):
     np.random.seed(1337)
     params = np.random.rand(n_params)
 
-    qnode_gpu = qml.QNode(circuit, dev_gpu, diff_method="adjoint")
+    qnode_gpu = qml.QNode(circuit, dev_mpi, diff_method="adjoint")
```

Comment:
Maybe
```suggestion
    qnode_mpi = qml.QNode(circuit, dev_mpi, diff_method="adjoint")
```
?
You don't need to, though, this is just for consistency.